.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
hmm ... this will make it hard to merge it back to upstream if we actually want to do this one day

Reply:
It would. It is very abandoned upstream, though. No commits since 2016, open issues/PRs with no feedback since 2017.